<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 44, 52);

font-family:&quot;Segoe UI&quot;;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="Left_Frame">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(33, 37, 43);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <widget class="QPushButton" name="btn_exit_user">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(40, 44, 52);
border: 2px solid rgb(40, 44, 52);
border-radius: 15px;
}

QPushButton:hover{
color:rgb(192, 218, 255);
background-color: rgb(43, 51, 67);  
border: 2px solid rgb(33, 37, 43);
border-radius: 15px;
}

QPushButton:pressed{
color:rgb(192, 218, 255);
background-color: rgb(40, 44, 52); 
border: 2px solid rgb(192, 218, 255);
border-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/Resource.qrc">
         <normaloff>:/icons/cil-account-logout.png</normaloff>:/icons/cil-account-logout.png</iconset>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="Right_Frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="Top_Frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>576</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(40, 44, 52);
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <property name="spacing">
           <number>40</number>
          </property>
          <property name="leftMargin">
           <number>40</number>
          </property>
          <property name="topMargin">
           <number>40</number>
          </property>
          <property name="rightMargin">
           <number>40</number>
          </property>
          <property name="bottomMargin">
           <number>40</number>
          </property>
          <item>
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(33, 37, 43);
border: 2px solid rgb(33, 37, 43);
border-radius: 15px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="H_lay_Firstname">
               <item>
                <widget class="QLabel" name="label_Firstname">
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(192, 218, 255);</string>
                 </property>
                 <property name="text">
                  <string>имя</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="edit_Firstname">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>47</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(192, 218, 255);
border: 2px solid rgb(78, 175, 255);  
border-radius: 15px;
</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="H_lay_Lastname">
               <item>
                <widget class="QLabel" name="label_Lastname">
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(192, 218, 255);</string>
                 </property>
                 <property name="text">
                  <string>фамилия</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="edit_Lastname">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>47</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(192, 218, 255);
border: 2px solid rgb(78, 175, 255);  
border-radius: 15px;
</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="H_lay_age">
               <item>
                <widget class="QLabel" name="label_age">
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(192, 218, 255);</string>
                 </property>
                 <property name="text">
                  <string>возраст</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="edit_age">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>47</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(192, 218, 255);
border: 2px solid rgb(78, 175, 255);  
border-radius: 15px;
</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="H_lay_phone">
               <item>
                <widget class="QLabel" name="label_phone">
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(192, 218, 255);</string>
                 </property>
                 <property name="text">
                  <string>номер телефона</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="edit_phone">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>47</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(192, 218, 255);
border: 2px solid rgb(78, 175, 255);  
border-radius: 15px;
</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="H_lay_mark">
               <item>
                <widget class="QLabel" name="label_mark">
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(192, 218, 255);</string>
                 </property>
                 <property name="text">
                  <string>марка машины</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="edit_mark">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>47</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(192, 218, 255);
border: 2px solid rgb(78, 175, 255);  
border-radius: 15px;
</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="H_lay_desription">
               <item>
                <widget class="QLabel" name="label_description">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(192, 218, 255);</string>
                 </property>
                 <property name="text">
                  <string>описание
проблемы</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="edit_description">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>47</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(192, 218, 255);
border: 2px solid rgb(78, 175, 255);  
border-radius: 15px;
</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="H_lay_data">
               <item>
                <widget class="QLabel" name="label_data">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(192, 218, 255);</string>
                 </property>
                 <property name="text">
                  <string>дата</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="dateEdit_user">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(192, 218, 255);
background-color: rgb(40, 44, 52);
border: 2px solid rgb(43, 51, 67); 
border-radius: 15px;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="Info_Frame">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(33, 37, 43);
border: 2px solid rgb(33, 37, 43);
border-radius: 15px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_Firstname_edit">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(192, 218, 255);</string>
               </property>
               <property name="text">
                <string>имя</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Lastname_edit">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(192, 218, 255);</string>
               </property>
               <property name="text">
                <string>фамилия</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_age_edit">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(192, 218, 255);</string>
               </property>
               <property name="text">
                <string>возраст</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_phone_edit">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(192, 218, 255);</string>
               </property>
               <property name="text">
                <string>номер телефона</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="Bot_Frame">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(40, 44, 52);
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="Holow_L_Frame">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(40, 44, 52);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QFrame" name="Save_Frame">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(40, 44, 52);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item alignment="Qt::AlignHCenter|Qt::AlignTop">
              <widget class="QPushButton" name="btn_save">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
color:rgb(192, 218, 255);
background-color: rgb(33, 37, 43);;  
border: 2px solid rgb(33, 37, 43);
border-radius: 15px;
}

QPushButton:hover{
color:rgb(192, 218, 255);
background-color: rgb(43, 51, 67);  
border: 2px solid rgb(33, 37, 43);
border-radius: 15px;
}

QPushButton:pressed{
color:rgb(192, 218, 255);
background-color: rgb(33, 37, 43);;  
border: 2px solid rgb(192, 218, 255);
border-radius: 15px;
}

</string>
               </property>
               <property name="text">
                <string>сохранить</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="Holow_R_Frame">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/Resource.qrc"/>
  <include location="../resources/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
